bindings/src/test/java/io/pravega/storage/extendeds3/S3ClientMock.java

```diff
@@ -86,6 +87,15 @@ public ListObjectsResult listObjects(String bucketName, String prefix) {
         return s3Impl.listObjects(bucketName, prefix);
     }
 
+    @Override
```

Comment:
list object needs to return paginated results.
For now it may be okay , but we need to fix this to implement pagination.

Reply:
This is the same behavior as in the defunct `S3FileSystemImpl.java`.